promql/testdata/duplicatelabel.test

```diff
@@ -0,0 +1,8 @@
+# Test duplicate labelset in promql output.
```

Comment:
Let's not create files with just one test, we can combine the few of these into one evaluation.test or similar.

Reply:
Done.